src/main/java/com/stripe/model/HasTypeData.java

```diff
@@ -0,0 +1,8 @@
+package com.stripe.model;
+
+import java.util.Map;
+
+public interface HasTypeData {
```

Comment:
Good call on the interface!

I wonder if we should call this `HasSourceTypeData` or something too? (Again, will likely always be sources specific.)